spring-integration-sftp/src/main/java/org/springframework/integration/sftp/support/GeneralSftpException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.sftp.support;
+
+import org.springframework.core.NestedRuntimeException;
+
+/**
+ * Simple runtime exception to wrap an SftpException.
+ *
+ * @author Gary Russell
+ * @since 4.3
+ *
+ */
+@SuppressWarnings("serial")
+public class GeneralSftpException extends NestedRuntimeException {
```

Comment:
M-m-m. I mean TODO in the `SftpSession`:
```
catch (SftpException e) {
	// TODO: in 5.0 remove e.toString() INT-3913
	throw new NestedIOException("Failed to remove file: " + e.toString(), e);
}
```

Or no... That is `IOException` which isn't `Runtime`, of course...